lib/Slic3r/GUI/Plater.pm

```diff
@@ -2510,12 +2510,15 @@ sub reload_from_disk {
             $Slic3r::GUI::Settings->{_}{reload_behavior} = $reload_behavior;
             $save = 1;
         }
-        if ($dlg->GetHideOnNext) {
-            $Slic3r::GUI::Settings->{_}{reload_hide_dialog} = 1;
+        if (!$dlg->GetHideOnNext) {
+            $Slic3r::GUI::Settings->{_}{reload_behavior} = 'prompt';
             $save = 1;
         }
         Slic3r::GUI->save_settings if $save;
         $dlg->Destroy;
+    } else {
```

Comment:
I added this to avoid one of the warnings (the rest of the code assumes that the value of `$reload_behavior` is numeric).